src/index.ts

```diff
@@ -714,6 +735,10 @@ class NixInstallerAction extends DetSysAction {
           dir: "/tmp",
           readOnly: false,
         },
+        {
+          dir: "/usr",
+          readOnly: true,
```

Comment:
I assume `readOnly` just means "writes to this directory will not propagate back to the host" and not "writes to this directory are not allowed, correct?

Reply:
Writes to that directory are prohibited/blocked/prevented. It is basically a read-only bind mount. This is after the install runs, though, so determinate-nixd is already on disk.

Reply:
Oh, that was my concern. Gotcha.